### ISSUE TITLE
About implementation of MBRBlock

### ISSUE THREAD

**Issue by TonyWangX** [bwang514/PerformanceNet#3]
Dear authors,

Thank you for the wonderful paper and code. I just to confirm the implementation of MBRBlock.

https://github.com/bwang514/PerformanceNet/blob/f2273f0ecfb090a18715a0bd08d50751e794e32e/src/model.py#L150-L151

I think these two lines intend to save the output from the conv-instancenorm block into **bands** and add **bands** to **x**. 

However, if I correctly understand torch.0.4.0 doc (https://pytorch.org/docs/0.4.0/torch.html?highlight=torch%20add#torch.add),  I think the code "torch.add(bands[i],1,t)" will not save **t** into **bands**.  

Since **bands** originally stores the value of **x**, this line "x = torch.add(x,1,torch.cat(bands, dim = 1))" will do nothing but simply add **x** with itself (well, not the same object but the same value).  As the result, what MBRBlock returns becomes **x+x**.

To testify that, I debugged and stopped at line 151 before executing it, then I compared the values in **x** and **bands**

> (Pdb) p torch.std(x - torch.cat(bands, dim = 1))
>tensor(0., device='cuda:0')
> (Pdb) p torch.mean(x - torch.cat(bands, dim = 1))
>tensor(0., device='cuda:0')

This indicates that **bands** stores the same value as **x**

With the two lines above, these four lines only multiplies x by 16, without using any conv in the MBRBlock. I can confirm this 
through debugging .
https://github.com/bwang514/PerformanceNet/blob/f2273f0ecfb090a18715a0bd08d50751e794e32e/src/model.py#L214-L217

So, could you check the implementation of MBRBlock? 

FYI
I installed dependency according to requirement.txt except scikt-learn and pretty_midi. 
But I think these two packages are not related to the above issue.

**Comment by bwang514** [comment]
Hi Xin,

Thanks for pointing this out and I have fixed the bug.

Bryan
